### PR TITLE
add Hazelcast session manager

### PR DESCRIPTION
**Pull request by olamy**
Signed-off-by: olivier lamy <olamy@webtide.com>

### CODE REVIEW

**Comment by olamy** [comment]
do not merge as I still have the documentation to add.

**Comment by olamy** [comment]
documentation page added.
Please review @janbartel  @WalkerWatch

File: jetty-hazelcast/src/main/config/modules/session-store-hazelcast-remote.mod [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+annotations
+webapp
+
+[files]
+maven://com.hazelcast/hazelcast-all/3.8.2|lib/hazelcast/hazelcast-all-3.8.2.jar
+maven://com.hazelcast/hazelcast-jetty9-sessionmanager/1.0.2|lib/hazelcast/hazelcast-jetty9-sessionmanager-1.0.2.jar
+maven://org.eclipse.jetty/jetty-nosql/${jetty.version}|lib/hazelcast/jetty-nosql-${jetty.version}.jar
+
+[xml]
+etc/sessions/hazelcast/default.xml
```
Comment (janbartel) [inline jetty-hazelcast/src/main/config/modules/session-store-hazelcast-remote.mod]: Should that be etc/sessions/hazelcast/remote.xml ??

File: jetty-hazelcast/src/main/config/modules/session-store-hazelcast-embedded.mod [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#
```
Comment (janbartel) [inline jetty-hazelcast/src/main/config/modules/session-store-hazelcast-embedded.mod]: I think I'd prefer not to confuse users by bringing in 9.4 terminology like session-store, so can you rename the module to maybe just hazelcast-embedded-sessions.mod or something?

File: jetty-hazelcast/src/main/config/modules/session-store-hazelcast-remote.mod [hunk 3]
```diff
@@ -0,0 +1,26 @@
+#
```
Comment (janbartel) [inline jetty-hazelcast/src/main/config/modules/session-store-hazelcast-remote.mod]: See comment above for session-store-hazelcast-embedded.mod.

File: jetty-hazelcast/src/main/config/modules/session-store-hazelcast-embedded.mod (hunk 2 above)
Comment (olamy) [inline jetty-hazelcast/src/main/config/modules/session-store-hazelcast-embedded.mod]: yup looks good.

File: jetty-hazelcast/src/main/config/modules/session-store-hazelcast-remote.mod (hunk 1 above)
Comment (olamy) [inline jetty-hazelcast/src/main/config/modules/session-store-hazelcast-remote.mod]: yup good catch.

**Comment by olamy** [comment]
ping. I think we are done here.
Someone to click the merge button? 
Or let me know if still issues pending.

**Pull request merged**